qa/L0_storage_azure/test.sh

```diff
@@ -110,21 +110,15 @@ sleep 10
 
 # Test 1 Scenarios:
 # 1. access blob using shared key in envs
-# 2. adding more scenarios in future 
+# 2. adding more scenarios in future
 for ENV_VAR in "shared_key"; do
     SERVER_LOG=$SERVER_LOG_BASE.$ENV_VAR.log
     CLIENT_LOG=$CLIENT_LOG_BASE.$ENV_VAR.log
     MODEL_REPO="${AS_URL}/models"
-    if [ "$ENV_VAR" == "sas" ]; then
```

Comment:
(bumping this comment so it doesn't get missed)

@bnookala please keep this section in the test script